src/ExtensionsSample/packages.config

```diff
@@ -2,7 +2,7 @@
 <packages>
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net45" />
-  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.6.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.6.5-alpha" targetFramework="net45" />
```

Comment:
You also need to update your package nuspec - it still references version 0.6.2. Don't you now need the min version to be the new alpha version?